index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <!-- <p>My Logo</p> -->
+        <div>
+            <span>My Logo</span>
+            <!-- <img class="navbar" src="images/menu.png" alt="navigation"> -->
+            <img src="images/menu.png" alt="navigation">

```

Comment:
- [ ] This is a mobile menu button. Therefore, it would be more semantically appropriate to wrap it with a `<nav>` tag.